helm/libero-editor/templates/deployment-article-store.yaml

```diff
@@ -36,3 +36,31 @@ spec:
           readinessProbe:
             tcpSocket:
               port: http
+          env:
+            - name: DB_URL
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.database.secret | quote }}
+                  key: url
+            - name: DB_NAME
+              value: {{ .Values.database.name | quote }}
+            - name: DB_SSL_VALIDATE
+              value: {{ .Values.database.sslValidate | quote }}
+            - name: AWS_REGION
+              value: {{ .Values.aws.region | quote }}
+            - name: AWS_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.aws.secret | quote }}
```

Comment:
Are `AWS_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY` supposed to be the same? The are both set to the value of `Values.aws.secret`?

Reply:
They're related so they live in the same kubeseal file. `Values.aws.secret` references the file and the key points to the value inside